uint16 inchworm_idx

---

inchworm_algo/InchwormMsg state